_pages/user-experience/knowledge-articles.md

```diff
@@ -15,9 +15,54 @@ sidenav:
     href: "user-experience/failure-proof/"
   - text: Knowledge articles
     href: "user-experience/knowledge-articles/"
-  - text: FAQ content 
+  - text: FAQ content
     href: "user-experience/faq-content/"
 
 ---
 
-## Knowledge Articles
+## Prepare customer support materials
+
+As you prepare to launch your integration with Login.gov, it is important to consider how your agency will support new users. Here are some suggestions to consider:
+
+<ul>
+  <li>
+    Announce the upcoming change on your site and include a time frame rather than exact date.
+  </li>
+  <li>
+    Link to the <a
+    class="usa-link usa-link--external"
+    rel="noreferrer"
+    target="_blank"
+    href="https://www.login.gov/create-an-account/"
+    >Create an account</a>
+    page at Login.gov so users can read more about creating an account.
+  </li>
+  <li>
+    Include contact information for your help desk or Login.gov customer support on a dedicated help page.
```

Comment:
Suggest updating to "Include contact information for your agency's help desk or a link to the Login.gov help center [https://www.login.gov/help/] on a dedicated page." 
Specify the help center to avoid partners posting the phone number.

Reply:
added in https://github.com/18F/identity-dev-docs/pull/415/commits/1d6ff1556f48f806778561a551a99accd70f2e9c